docs/lib/datastore/fragments/flutter/getting-started/40_codegen.md

```diff
@@ -0,0 +1 @@
+Coming Soon. Please use Option #2 for now.
```

Comment:
add link to option 2 (I know it's just below but will help clarify first time readers)

Reply:
Done.